plugins/Name_Multilingual.py

```diff
@@ -348,3 +384,24 @@ class father:
 
         assert not self.p.way(None, {"name": u"Avenue جادة", "name:fr": u"Avenue", "name:ar": u"جادة"}, None)
         assert not self.p.way(None, {"name": u"Avenue 17 / جادة 17", "name:fr": u"Avenue 17", "name:ar": u"جادة 17"}, None)
+
+    def test_eu(self):
+        TestPluginCommon.setUp(self)
+        self.p = Name_Multilingual(None)
+        class _config:
+            options = {"language": ["es", "eu"], "multilingual_style": "sp_eu"}
+        class father:
+            config = _config()
+        self.p.father = father()
+        self.p.init(None)
+
+        assert not self.p.way(None, {"name": "Carretera Ollaretxe errepidea", "name:es": "Carretera Ollaretxe", "name:eu": "Ollaretxe errepidea"}, None)
+        assert not self.p.way(None, {"name": "Kale Nagusia / Calle Mayor", "name:es": "Calle Mayor", "name:eu": "Kale Nagusia"}, None)
+        assert not self.p.way(None, {"name": "Vicente Blasco Ibañez kalea / Calle Vicente Blasco Ibáñez", "name:es": "Calle Vicente Blasco Ibáñez", "name:eu": "Vicente Blasco Ibañez kalea"}, None)
+        assert not self.p.way(None, {"name": "Calle San Diego kalea", "name:es": "Calle San Diego", "name:eu": "San Diego kalea"}, None)
+        assert not self.p.way(None, {"name": "Calle Islas Canarias / Kanariar Uharteen kalea", "name:es": "Calle Islas Canarias", "name:eu": "Kanariar Uharteen kalea"}, None)
+        assert not self.p.way(None, {"name": "Vicente Blasco Ibañez kalea / Calle Vicente Blasco Ibáñez", "name:es": "", "name:eu": ""}, None)
+        assert self.p.way(None, {"name": "Calle Islas Canarias / Kanariar Uharteen kalea", "name:es": "Calle Canarias", "name:eu": "Kanarias kalea"}, None)
+        assert self.p.way(None, {"name": "Calle San Diego", "name:es": "", "name:eu": "San Diego kalea"}, None)
+        assert self.p.way(None, {"name": "Kale Nagusia", "name:es": "Calle Mayor", "name:eu": ""}, None)
```

Comment:
```suggestion
        assert self.p.way(None, {"name": "Calle San Diego"}, None)
        assert self.p.way(None, {"name:eu": "San Diego kalea"}, None)
        assert self.p.way(None, {"name:es": "Calle Mayor"}, None)
        assert self.p.way(None, {"name": "Calle San Diego", "name:eu": "San Diego kalea"}, None)
        assert self.p.way(None, {"name": "Calle San Diego", "name:es": "", "name:eu": "San Diego kalea"}, None)
        assert self.p.way(None, {"name": "Kale Nagusia", "name:es": "Calle Mayor"}, None)
        assert self.p.way(None, {"name": "Kale Nagusia", "name:es": "Calle Mayor", "name:eu": ""}, None)
```

Reply:
I'd say the failing test come from this change. As I understand, it's not the same, since what I was testing in the first clause is the presence of `name` and `name:eu`, with different values, and not `name:es`. And similar for the second one: `name` and `name:es` different, and no `name:eu`

Reply:
No sure to understand your response. The function should not fails.

Reply:
Look at the [failing job](https://github.com/osm-fr/osmose-backend/actions/runs/15054880334/job/42318339779?pr=2484): it fails just for these changes you made. Undo these changes and letś try again.

AFAI understand you're not testing the same case I did, aren't you? I wanted to test the cases I before-commented: two tags, name and name:lang, different values each other. It should warn the user "hey, something is missing here"

Reply:
I remove when there is only `name`.

The code should not crash, even if there is nothing to be done.